src/bzfs/bzfsAPI.cxx

```diff
@@ -4302,6 +4321,11 @@ BZF_API void bz_reloadMasterBans()
 
     // reload the banlist
     logDebugMessage(3,"Reloading master bans\n");
+
+    bz_ReloadData_V1 event;
+    event.target = "masterbans";
+    worldEventManager.callEvents(bz_eReloadEvent, &event);
```

Comment:
this method can short-circuit (line 4320 above), meaning the event manager will not dispatch this message. Is that right?